test/MyTested.AspNetCore.Mvc.ViewComponents.Test/BuildersTests/InvocationsTests/ShouldHaveTests/ShouldHaveViewComponentAttributesTests.cs

```diff
@@ -89,5 +138,40 @@ public void AttributesShouldThrowEceptionWithActionContainingNumberOfAttributesT
                 },
                 "When testing AttributesComponent was expected to have 1 attribute, but in fact found 2.");
         }
+
+        [Fact]
+        public void AndAlsoShouldWorkCorrectly()
+        {
+            MyViewComponent<PocoViewComponent>
+                .Instance()
+                .InvokedWith(c => c.Invoke())
+                .ShouldHave()
+                .NoAttributes()
+                .AndAlso()
+                .ShouldPassForThe<PocoViewComponent>((viewComponent) =>
+                {
+                    Assert.NotNull(viewComponent);
+                    Assert.Empty(viewComponent.TempData);
+                });
+        }
+
+        [Fact]
```

Comment:
I believe this test is not needed. We are essentially testing here that the provided Assert.True method from xunit is working correctly. Or maybe I am missing the point?